lib/cli/test/fixtures/riot/package.json

```diff
@@ -12,6 +12,7 @@
     "riot": "^3.11.2"
   },
   "devDependencies": {
+    "cross-env": "^5.2.0",
```

Comment:
Does it mean that user should install it before running `getstorybook`?

Reply:
Yes, see this package.json. It's something the user is supposed to consciously configure and use. Not something that should be a dependency of storybook package.

Reply:
So storybook doesn't work without it? Why? We should fix or at least document it.

Reply:
No, storybook works without it. I am saying this is something people use in their web apps which the fixture here is.

Reply:
> No, storybook works without it

Let's not add it to the fixture without need then

Reply:
`vue-cli` doesn't add it. I don't know why this fixture has it. You can see it here, https://github.com/storybooks/storybook/pull/4209/files#diff-a3134227a935db48acff355fe7fbedfdR8